admin/includes/languages/english/category_product_listing.php

```diff
@@ -68,15 +68,18 @@
 // From categories.php in 1.5.5
 // categories status
 define('TEXT_INFO_HEADING_STATUS_CATEGORY', 'Change Category Status for:');
-define('TEXT_CATEGORIES_STATUS_INTRO', 'Change the Category Status to: ');
-define('TEXT_CATEGORIES_STATUS_OFF', 'OFF');
-define('TEXT_CATEGORIES_STATUS_ON', 'ON');
-define('TEXT_PRODUCTS_STATUS_INFO', 'Change ALL Product Status to: ');
-define('TEXT_PRODUCTS_STATUS_OFF', 'OFF');
-define('TEXT_PRODUCTS_STATUS_ON', 'ON');
+define('TEXT_CATEGORIES_STATUS_INTRO', 'Change <strong>this</strong> category\'s status to: ');
+define('TEXT_CATEGORIES_STATUS_OFF', 'Disabled');
+define('TEXT_CATEGORIES_STATUS_ON', 'Enabled');
+define('TEXT_PRODUCTS_STATUS_INFO', 'Change ALL included products status to:');
```

Comment:
Shouldn't a note about linked product be included or did I miss something in code review where the text of say `TEXT_CATEGORIES_STATUS_WARNING` is applied?

Reply:
define('TEXT_CATEGORIES_STATUS_WARNING', '<strong>WARNING ...</strong><br />Note: Disabling a category will disable all products in this category. Linked products located in this category that are shared with other categories will also be disabled.');

Since this PR makes this result visible and an option, I don't think the warning is needed.